### PR TITLE
Allow laravel/serializable-closure 2

### PR DESCRIPTION
**Pull request by simoheinonen**
| Q             | A
| ------------- | ---
| Branch        | master for features and deprecations
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Documented?   | no
| Fixed tickets |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "amphp/parallel": "^2.1",
         "doctrine/collections": "^1.6.8 || ^2.0",
         "gitonomy/gitlib": "^1.3",
-        "laravel/serializable-closure": "^1.1",
+        "laravel/serializable-closure": "^1.1 || ^2.0",
```
Comment (veewee) [inline composer.json]: Is it worth supporting both? Supporting 2.x seems like a better approach to me - but I'm not sure how heavily people depend on this dependency in Laravel. Any clue?

**Review by veewee — COMMENTED** [review]
Thanks for the PR!

File: composer.json (hunk 1 above)
Comment (simoheinonen) [reply composer.json]: No idea :D 

I guess the biggest (or only) difference between 1 and 2 is they dropped PHP 7.x support, which GrumPHP doesn't support anyway so probably fine to go just with 2

**Comment by mrunkel** [comment]
Any idea/plan when this will get "released" and available on packagist?

To add some context, this is preventing the installation of grumphp in laravel 11 project.

**Comment by veewee** [comment]
@mrunkel Forgot about this one. I'll tag it somewhere this week.

**Comment by veewee** [comment]
Tagged ! (Finally ...) :)

**Pull request merged**